downstream/assemblies/eda/assembly-installation-eda-controller.adoc

```diff
@@ -11,6 +11,11 @@ Similar to the {ControllerName} and {HubName} components, the setup for {EDAcont
 {EDAcontroller} must be installed on a separate server and cannot be installed on the same host as {HubName} and {ControllerName}.
 ====
 
+[NOTE]
+====
+If you plan to run {EDAName} on {RHEL} 8, you must have cgroup v2 enabled on your system to set memory limits. For specific instructions, see the Knowledge-Centered Support (KCS) article, link:https://access.redhat.com/solutions/7054905[Ansible Automation Platform Event-Driven Ansible controller for RHEL 8 requires cgroupv2].
```

Comment:
cgroup v2 is optional, only required if you want to set memory limits. So, the text should be more like:

```suggestion
If you plan to run {EDAName} on {RHEL} 8 with memory limits, you must have cgroup v2 enabled on your system previously. For specific instructions, see the Knowledge-Centered Support (KCS) article, link:https://access.redhat.com/solutions/7054905[Ansible Automation Platform Event-Driven Ansible controller for RHEL 8 requires cgroupv2].
```
Otherwise it suggest that this step is mostly mandatory, don't?

Reply:
I think I understand what you're saying, but using "previously" might lead to more questions. How about the following: 

```suggestion
If you plan to run {EDAName} on {RHEL} 8, you must have cgroup v2 enabled on your system to set memory limits before you install {EDAName}. For specific instructions, see the Knowledge-Centered Support (KCS) article, link:https://access.redhat.com/solutions/7054905[Ansible Automation Platform Event-Driven Ansible controller for RHEL 8 requires cgroupv2].
```

Reply:
The style of the text is your domain :)
I'm just concerned that the statement `If you plan to run {EDAName} on {RHEL} 8, you must have cgroup v2 enabled on your system to set memory limits before you install {EDAName}.` to me means that It's a mandatory step because EDA needs to set memory limits.  My point is to keep clear that this step is optional and only required if you want to set memory limits, but you can skip it and install EDA without that capability.

Reply:
@Alex-Izquierdo - Please review the updates, per your explanation above and let me know if this is clearer.

Reply:
Much better now IMO. Thanks

Reply:
Thanks for your review, @Alex-Izquierdo. I will proceed with the next steps in the docs review process.